CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/ErrorReportSessionData.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2010-2023 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.errorreport;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+/**
+ * This is a data storage class to maintain the list of errors that have been
+ * generated, and the list of errors that have been displayed in the Error
+ * Report View
+ *
+ * @author OrionsGuardian
+ */
+public class ErrorReportSessionData {
+
+    final private ArrayList<ErrorReportEntry> sessionErrors = new ArrayList<>();
+    final private ArrayList<ErrorReportEntry> displayedErrors = new ArrayList<>();
+
+    private static ErrorReportSessionData instance = null;
+    private static Double nextEntryId = 0D;
+    public static Date lastUpdate = new Date();
+
+    public static boolean screenUpdateRequested = false;
+
+    ErrorReportSessionData() {
+    }
```

Comment:
More just a check, did you want this to be package-private? If it can be public, you can remove, else may be good to make protected or private